tools/bag_processing/scripts/convert_bayer_bag.py

```diff
@@ -74,21 +97,36 @@ def convert_bayer_to_grayscale(
         default="/mgt/img_sampler/nav_cam/image_record",
         help="Output gray image topic.",
     )
+    parser.add_argument(
+        "-c",
+        "--color-image-topic",
+        default="/hw/cam_nav/image_color",
+        help="Output color image topic.",
+    )
```

Comment:
It might be a cleaner interface to add the image topic and whether it needs to be converted as separate options.  For the latter, you can use a bool switch:
`    parser.add_argument(
        "--save-color-image",
        dest="save_color_images",
        action="store_true",
        help="Save color images to bagfile.",
    )`

If the action is store_true, it default stores false and vice versa.

Reply:
What I did now was if we set an empty topic, to ignore it, so:
````
rosrun bag_processing convert_bayer_bag.py 2022-02-28-18-46-30.bag -s -k -c ""
````
does not record color image

Reply:
Right, I was thinking to avoid the empty parenthesis, as the bool switch is probably a more common usage pattern. The difficulty though is coming up with different abbreviations for saving a color/grayscale image vs. setting the color/grayscale image topic.  My guess is the topics won't be changed as often, so reserving the -c/g abbreviation for saving the color/grayscale images and not having an abbreviation for the topics might be ok

Reply:
yeah, that's actually the reason I didn't add separate parameters for it, I started doing it and couldn't come up with names lol. Maybe use caps, like disable is -C and -G?

Reply:
Oh, that's already taken care of with the bool switch - 
`script -s` 
enables s and 
`script` 
runs without s.  We can add a comment in the parameter description to make that more clear, like 
`Default does not perform color conversion`
I'm actually curious now if arg parse is smart enough to say something like that, our text setting displays default values.

Reply:
got it, I didn't read your comment properly earlier xD

Reply:
Hi, Marina- I like having these bool flags but suggest they should be called "--no-color" and "--no-gray" instead of "-c" and "-g". For whatever reason, I have a strong prior that "-c" should *enable* color so it hurts my brain to have it negated like that.